rm/rm.h

```diff
@@ -62,23 +61,25 @@
  *
  * A resource (m0_rm_resource) is associated with various file system entities:
  *
- *     - file meta-data. Credits to use this resource can be thought of as locks
- *       on file attributes that allow them to be cached or modified locally;
+ *   - file meta-data. Credits to use this resource can be thought of as locks
+ *     on file attributes that allow them to be cached or modified locally;
  *
- *     - file data. Credits to use this resource are extents in the file plus
- *       access mode bits (read, write);
+ *   - file data. Credits to use this resource are extents in the file plus
+ *     access mode bits (read, write);
  *
- *     - free storage space on a server (a "grant" in Lustre
- *       terminology). Credit to use this resource is a reservation of a given
- *       number of bytes;
+ *   - free storage space on a server (a "grant" in Lustre
+ *     terminology). Credit to use this resource is a reservation of a given
+ *     number of bytes;
```

Comment:
What is being fixed in these formatting changes?

Reply:
Doxygen-generated doc generates fixed-style text for these items instead of normal bulletted list without this change.